[ADD] Localizations: Functional documentation for l10n_cl_edi

ren-odoo opened the pull request:
Functional guidance for the Chilean localization:
- Electronic Invoice configuration 
- Use cases and testing
- Financial Reports

jubodoo commented:
@jcs-odoo If you have tim eto test it tomorrow would be great

ren-odoo commented:
Hi @jcs-odoo 

Were you able to take a look at the documentation? We have several customers waiting for it. 

Thank you in advance. 

cc. @jubodoo

AntoineVDV commented:
> Were you able to take a look at the documentation? We have several customers waiting for it.

Hey @ren-odoo, with the OXP approaching I doubt that the content team will have the time to review this before two weeks. If you asked JCS for a doc review let me know and I'll handle it. If you're asking for an accounting review, I'm afraid that you'll have to wait :p

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Install the Chilean localization modules
+----------------------------------------
+For this, go to *Apps* and search for Chile. Then click *Install* in the module Chile 
+E-invoicing, this module has a dependency with Chile - Accounting in case this last
+one is not installed, Odoo installs it automatically with E-invoicing.
```
```suggestion
For this, go to *Apps* and search for Chile. Then click *Install* in the module **Chile 
E-invoicing**. This module has a dependency with **Chile - Accounting**. In case this last
one is not installed, Odoo installs it automatically with E-invoicing.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+Introduction
+============
+
+The Chilean localization has been improved and extended in Odoo v13,in this version the next 
+modules are available:
```
```suggestion
The Chilean localization has been improved and extended in Odoo v13. In this version, the next 
modules are available:
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+----------------
+Once the modules are installed, the first step is to set up your company data. Additional
+to the basic information, you need to add all the data and elements required for Electronic 
+Invoice, the easiest way to configure it is in 
+:menuselection:`Accounting --> Settings --> Chilean Localization`
```
```suggestion
Invoice, the easiest way to configure it is in 
:menuselection:`Accounting --> Settings --> Chilean Localization`.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Configure DTE Incoming email server
+***********************************
+In order to receive you the claim and acceptance emails from your customers it is
+crucial to define the DTE incoming email server, considering this configuration: 
```
```suggestion
In order to receive you the claim and acceptance emails from your customers, it is
crucial to define the DTE incoming email server, considering this configuration: 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+   
+Certificate
+~~~~~~~~~~~
+In order to generate the electronic invoice signature, a digital certificate with extension(.pfx)
+is required, proceed to this section and load your file and password. 
```
```suggestion
In order to generate the electronic invoice signature, a digital certificate with the extension ``.pfx``
is required, proceed to this section and load your file and password. 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+.. image:: media/Chile04.png
+   :align: center
+
+A certificate can be shared between several users, if this is the case leave the user field empty,
+so all your billing users can use the same one. On the other hand,  if you need to restrict the
+certificate for a particular user, just define the users in the certificate. 
+
```
```suggestion
A certificate can be shared between several users. If this is the case leave the user field empty,
so all your billing users can use the same one. On the other hand,  if you need to restrict the
certificate for a particular user, just define the users in the certificate. 

```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+   
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+The chart of accounts is installed by default as part of the set of data included in
+the localization module, the accounts are mapped automatically in: 
```
```suggestion
The chart of accounts is installed by default as part of the set of data included in
the localization module. The accounts are mapped automatically in: 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+~~~~~~~
+Identification Type and VAT
+***************************
+As part of the Chilean localization, the identification types defined by the SII
+are now available on the Partner form, this information is essential for most transactions.
```
```suggestion
As part of the Chilean localization, the identification types defined by the SII
are now available on the Partner form. This information is essential for most transactions.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+E-invoicing, this module has a dependency with Chile - Accounting in case this last
+one is not installed, Odoo installs it automatically with E-invoicing.
+
+.. image:: media/Chile01.png
+   :align: center
```
Ideally, add *alt* tags to all your images on the page.
Confer this [doc](https://www.odoo.com/documentation/user/13.0/contributing/documentation/content_guidelines.html#alt-tags).

For example
```suggestion
.. image:: media/Chile01.png
   :align: center
   :alt: Chilean module to install on Odoo.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+ 
+TaxpayerType
+************
+In Chile the document type associated with customers and vendors transactions is defined
+based on the Taxpayer Type, this field should be defined in the partner form, when creating
+a customer is important you make sure this value is set:
```
```suggestion
In Chile, the document type associated with customers and vendors transactions is defined
based on the Taxpayer Type. This field should be defined in the partner form, when creating
a customer is important you make sure this value is set:
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Taxes
+~~~~~
+As part of the localization module the taxes are created automatically with their related
+financial account and configuration.
```
```suggestion
As part of the localization module, the taxes are created automatically with their related
financial account and configuration.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+Chile has several tax types, the most common ones are:
+
+- VAT. Is the regular VAT and it can have several rates. 
+- ILA (Impuesto a la Ley de Alcholes). Taxes for alcoholic drinks, it has a different rate.
```
```suggestion
- ILA (Impuesto a la Ley de Alcholes). Taxes for alcoholic drinks. It has a different rate.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Fiscal Positions
+~~~~~~~~~~~~~~~~
+Based on the purchase transactions the VAT can have different affections, this will be done
+in Odoo using the default purchase fiscal positions. 
+
```
```suggestion
Based on the purchase transactions, the VAT can have different affections. This will be done
in Odoo using the default purchase fiscal positions. 

```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Document Types
+~~~~~~~~~~~~~~
+In some Latin America countries, including Chile, some accounting transactions like invoices and
+vendor bills are classified by  document types defined by the government fiscal authorities (In
+Chile case: SII).
```
```suggestion
In some Latin American countries, including Chile, some accounting transactions like invoices and
vendor bills are classified by document types defined by the government fiscal authorities (In
Chile case: SII).
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+vendor bills are classified by  document types defined by the government fiscal authorities (In
+Chile case: SII).
+
+The document type is an essential information that needs to be displayed in the printed reports and
+that needs to be easily identified, within the set of invoices as well of account moves.
```
```suggestion
The document type is essential information that needs to be displayed in the printed reports and
that needs to be easily identified within the set of invoices as well of account moves.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+The document type is an essential information that needs to be displayed in the printed reports and
+that needs to be easily identified, within the set of invoices as well of account moves.
+
+Each document type can have a unique sequence per journal where it is assigned. As part of the
+localization, the Document Type include the country on which the document is applicable and the data
+is created automatically when the localization module is installed.
```
```suggestion
Each document type can have a unique sequence per journal where it is assigned. As part of the
localization, the Document Type includes the country on which the document is applicable and the data
is created automatically when the localization module is installed.
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+*********
+The transactions purchase can be managed with a single journal, but sometimes companies use
+more than one in order to handle some accounting transactions that are not related to vendor
+bills but can be easily register using this model, for example:
```
```suggestion
bills but can be easily registered using this model, for example:
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+~~~~~~~~~~~~~
+When creating sales journals the next information must be filled in: 
+
+**Point of sale type**: If the Sales journal will be used for electronic documents the option
+Online must be selected, otherwise if the journal is used for invoices imported from a previous
+system or if you are using the SII portal “Facturación MiPyme“ you can use the option Manual. 
```
```suggestion
**Point of sale type**: If the Sales journal will be used for electronic documents, the option
Online must be selected. Otherwise, if the journal is used for invoices imported from a previous
system or if you are using the SII portal “Facturación MiPyme“ you can use the option Manual. 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+Online must be selected, otherwise if the journal is used for invoices imported from a previous
+system or if you are using the SII portal “Facturación MiPyme“ you can use the option Manual. 
+
+**Use Documents**: This field is used to define if the journal will use Document Types, it’s only
+applicable on Purchase and Sales journals which are the ones that can be related to the
+differentset of document types available in Chile. By default all the sales journals created will
+use documents. 
```
```suggestion
**Use Documents**: This field is used to define if the journal will use Document Types. It is only
applicable to Purchase and Sales journals that can be related to the
different sets of document types available in Chile. By default, all the sales journals created will
use documents. 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+segmentations you have in your journals (example per office) thus avoiding distributing a range
+of folios. In this way, each branch would have its private key of the respective CAF.
+
+Please refer to the SII documentation to check the detail how to acquire the CAF:
```
```suggestion
Please refer to the SII documentation to check the detail on how to acquire the CAF:
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+   :align: center
+
+
+As a result the sequence color changes to black an the column Next number and quantity
+available are updated:
```
```suggestion
As a result, the sequence color changes to black and the column Next number and quantity
available are updated:
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+  .. image:: media/Chile16.png
+     :align: center
+
+The DTE Status is updated automatically by Odoo with an scheduled action that runs every day
```
```suggestion
The DTE Status is updated automatically by Odoo with a scheduled action that runs every day
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+~~~~~~~~~~~~~~~~~~
+When the Invoice is created as a result of another fiscal document, the information related to the
+originator document must be registered in the Tab Cross Reference, which is commonly used for 
+credit or debit notes, but in some cases can be used on Customer Invoices as well. In case of the
+credit and debit notes it is set automatically by Odoo:
```
```suggestion
credit or debit notes, but in some cases can be used on Customer Invoices as well. In the case of the
credit and debit notes, they are set automatically by Odoo:
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Common Errors
+~~~~~~~~~~~~~
+There are multiple reasons behind a rejection from the SII, but these are some of the common erros
```
```suggestion
There are multiple reasons behind a rejection from the SII, but these are some of the common errors
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Credit Notes
+------------
+When a cancellation or correction is needed over a validated invoice, a credit note must be
+generated, it’s important to consider that a CAF file is required for the Credit Note sequence,
+which is identified as document 64 in the SII.  
```
```suggestion
When a cancellation or correction is needed over a validated invoice, a credit note must be
generated. It is important to consider that a CAF file is required for the Credit Note sequence,
which is identified as document 64 in the SII.  
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Cancel Referenced document
+**************************
+In case you need  to cancel or invalid a Invoice, use the button Add Credit note and select Full
```
```suggestion
In case you need  to cancel or invalid an Invoice, use the button Add Credit note and select Full
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+.. image:: media/Chile36.png
+   :align: center
+
+If you claim a vendor bill the status changes from draft to cancel automatically, considering this
+as best practice, all the Claim documents should be cancelled as they won’t be valid for your
+accounting records. 
```
```suggestion
If you claim a vendor bill, the status changes from draft to cancel automatically. Considering this
as best practice, all the Claim documents should be canceled as they won’t be valid for your
accounting records. 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+
+Propuesta F29
+-------------
+The form F29 is a new system that the SII enabled to taxpayers, and that replaces the Purchase and
+Sales Books. This report is integrated by Purchase Register (CR) and the Sales Register (RV)
+its purpose is to support the transactions realted to VAT improving its control and declaration. 
```
```suggestion
The form F29 is a new system that the SII enabled to taxpayers, and that replaces the Purchase and
Sales Books. This report is integrated by Purchase Register (CR) and the Sales Register (RV).
Its purpose is to support the transactions related to VAT, improving its control and declaration. 
```

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+As part of the Chilean localization, the identification types defined by the SII
+are now available on the Partner form, this information is essential for most transactions.
+
+.. image:: media/Chile06.png
+   :align: center
+   :scale: 25
```
Is the scale a way to make the image tinier?
Try to change the height instead. It should work

ren-odoo replied:
Image updated in the last commit.

jcs-odoo commented on accounting/fiscal_localizations/localizations/chile.rst:
```diff
@@ -0,0 +1,627 @@
+.. image:: media/Chile38.png
+   :align: center
+
+
+
```
[Seealso section?](https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#seealso)

ren-odoo replied:
> Hello!
> Good job with the rst 👍
> 
> * Ideally, please add _alt_ tags to your images as explained [here](https://www.odoo.com/documentation/user/13.0/contributing/documentation/content_guidelines.html#alt-tags).
> * Ideally, leave a blank line after all titles. For example:
> 
> ```
> Title 1
> =======
> 
> Paragraph here.
> ```
> 
> * Also, could you check the end of some sentences? It seems that there are a lot of sentences being separated by a comma, rather than ending them with a point (e.g. "Sentence 1, sentence 2." instead of "Sentence 1. Sentence 2.")
> * Other comments are directly put on the code.
> * Abreviations such as _it's_ are rather for spoken language. Ideally, write them _it is_ instead.
> 
> Have a good day
> 
> Best regards,
> Jonathan

@jcs-odoo Thank you for the Feedback and the corrections,  I committed you suggestions (sorry  I was doing it individually at the beginning).  In the last commit with I added the Alt information on the images (except from img 16 to 20, as I was having issues with the alt on the nested list, it wasn't working), I also added the blank line after the subtitles. 

Regards.

ren-odoo replied:
@jcs-odoo sorry but I don't get in this section it might add the Seealso section.

jcs-odoo reviewed (changes requested):
Hello!
Good job with the rst 👍 

- Ideally, please add *alt* tags to your images as explained [here](https://www.odoo.com/documentation/user/13.0/contributing/documentation/content_guidelines.html#alt-tags).

- Ideally, leave a blank line after all titles. For example:

```
Title 1
=======

Paragraph here.
```

- Also, could you check the end of some sentences? It seems that there are a lot of sentences being separated by a comma, rather than ending them with a point (e.g. "Sentence 1, sentence 2." instead of "Sentence 1. Sentence 2.")

- Other comments are directly put on the code.

- Abreviations such as *it's* are rather for spoken language. Ideally, write them *it is* instead.

Have a good day

Best regards,
Jonathan

jcs-odoo commented:
> > Were you able to take a look at the documentation? We have several customers waiting for it.
> 
> Hey @ren-odoo, with the OXP approaching I doubt that the content team will have the time to review this before two weeks. If you asked JCS for a doc review let me know and I'll handle it. If you're asking for an accounting review, I'm afraid that you'll have to wait :p

Fair point made here. I didn't check the actions described in Odoo.

ren-odoo commented:
> or a doc review let me know and I'll handle it. If you're asking for an accounting review, I'

I was looking for the Document review it self, the module was already reviewed and approved by the Accounting PO.  

cc. @jcs-odoo

ren-odoo commented:
> Hello!
> Good job with the rst 👍
> 
> * Ideally, please add _alt_ tags to your images as explained [here](https://www.odoo.com/documentation/user/13.0/contributing/documentation/content_guidelines.html#alt-tags).
> * Ideally, leave a blank line after all titles. For example:
> 
> ```
> Title 1
> =======
> 
> Paragraph here.
> ```
> 
> * Also, could you check the end of some sentences? It seems that there are a lot of sentences being separated by a comma, rather than ending them with a point (e.g. "Sentence 1, sentence 2." instead of "Sentence 1. Sentence 2.")
> * Other comments are directly put on the code.
> * Abreviations such as _it's_ are rather for spoken language. Ideally, write them _it is_ instead.
> 
> Have a good day
> 
> Best regards,
> Jonathan

Identification transaction for invoice and Status update

ren-odoo commented:
> Hello!
> Good job with the rst 👍
> 
> * Ideally, please add _alt_ tags to your images as explained [here](https://www.odoo.com/documentation/user/13.0/contributing/documentation/content_guidelines.html#alt-tags).
> * Ideally, leave a blank line after all titles. For example:
> 
> ```
> Title 1
> =======
> 
> Paragraph here.
> ```
> 
> * Also, could you check the end of some sentences? It seems that there are a lot of sentences being separated by a comma, rather than ending them with a point (e.g. "Sentence 1, sentence 2." instead of "Sentence 1. Sentence 2.")
> * Other comments are directly put on the code.
> * Abreviations such as _it's_ are rather for spoken language. Ideally, write them _it is_ instead.
> 
> Have a good day
> 
> Best regards,
> Jonathan

@jcs-odoo Thank you for the feedback, I committed your suggestions (sorry I did it individually at the beginning), in the last commit I added the alt-tag information for the images (except for images 16 to 20 as I was getting errors with the Nested list using the alt-g) the blank space was also added after the titles ans subtitles. 

Thank you.

jubodoo reviewed (approved):
Approved the changes since it was urgent and you tested the rest.
